plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/actions-dropdown-menu.tsx

```diff
@@ -2,34 +2,56 @@
  * External dependencies
  */
 import { DropdownMenu } from '@wordpress/components';
-import { moreVertical, external, plugins } from '@wordpress/icons';
+import { moreVertical } from '@wordpress/icons';
 import { __ } from '@wordpress/i18n';
 
-export default function ActionsDropdownMenu() {
+/**
+ * Internal dependencies
+ */
+import { Subscription } from '../../types';
+import { ADMIN_URL } from '../../../../../utils/admin-settings';
+
+export default function ActionsDropdownMenu( props: {
+	subscription: Subscription;
+} ) {
+	const controls = [
+		{
+			title: __( 'Manage in WooCommerce.com', 'woocommerce' ),
```

Comment:
"In" isn't necessarily wrong, but it's more natural to me to use "on" for a website – like "I saw it on Twitter".

```suggestion
			title: __( 'Manage on Woo.com', 'woocommerce' ),
```